concent_api/core/tests/utils.py

```diff
@@ -240,10 +242,14 @@ def _get_deserialized_task_to_compute(
     ) -> TaskToCompute:
 
         """ Returns TaskToCompute deserialized. """
+        if task_id is None:
```

Comment:
This change is not needed.

Reply:
it is - it enforces cohesiveness of task_id in `TaskToCompute`, `ComputeTaskDef` and `TaskHeader`.

Reply:
Right, I didn't seen the part below in this view.